src/components/dialogs/event-dialog.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import EventsTable from '../events-table';
+import { FormattedMessage } from 'react-intl';
+
+import {
+    Button,
+    Dialog,
+    DialogContent,
+    DialogTitle,
+    LinearProgress,
+    Typography,
+} from '@mui/material';
+import { Close } from '@mui/icons-material';
+
+const style = {
+    dialogTitleStyle: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+    },
+    dialogContentStyle: {
+        maxHeight: '70vh',
```

Comment:
Why forcing maxHeight on content, since Dialog component already has a scrolling system with static header and footer?

Reply:
Because the default maximum height of the dialog is too high and therefore when you have a large table the size of the dialog is too big

without maxHeight: '70vh',

![Capture d’écran de 2024-02-01 16-23-00](https://github.com/farao-community/gridcapa-app/assets/116628858/b17ccbb3-0d78-4d11-8c80-9cef763f91d1)

with maxHeight: '70vh',

![Capture d’écran de 2024-02-01 16-23-19](https://github.com/farao-community/gridcapa-app/assets/116628858/b8816ffa-4cbe-4ae1-883c-71b35e810b89)